### ISSUE TITLE
Counter-Strike 1.6 won't start

### ISSUE THREAD

**Issue by MihaiCristian** [ValveSoftware/steam-for-linux#905]
I wrote this post on Reddit and somebody recommended me to post it here instead.

I will just copy paste it, since you'll get the essence by reading it.

Alright, first of all I'm gonna start by saying that I'm a complete Linux noob. I've just recently started using it (due to school) and before that I only knew it existed, but nothing more than that. I'm using Ubuntu at the moment (they said it's one of the most noob-friendly, so there's that, don't judge). What I like about Linux is that even the smallest task requires some knowledge.

So, here I am, sadly, where I just hit a wall I can't pass. My laptop is a Lenovo Y570 with GT 555M video card.

What I've done so far:

I've installed CS 1.6 from Library. It works fine, but only with the integrated video card. So what I did was look for a driver for my Nvidia card. Aparently there is, I donwload it and installed it (here I learned about X Server and state levels ( I think that's how they're called) ). Sadly it doesn't work, after I reboot I'm stuck with a 640x480 desktop and I can't resize it. Also nvidia-current returns 'none', so I'm not sure if it's bugged or something. I unistall it (thanks Nvidia for nvidia-uninstall option).

I look again on the web and find out about Bumblebee. Looks promising, I download and install it. The problem I now get is :

```
[ERROR]Cannot access secondary GPU - error: [XORG] (EE) NVIDIA(0): Failed to initialize the NVIDIA GPU at PCI:1:0:0. Please

Error inserting bbswitch (/lib/modules/3.2.0-35-generic-pae/updates/dkms/bbswitch.ko): No ...
```

Turns out this bug is specific to my laptop: [1] https://bugzilla.kernel.org/show_bug.cgi?id=42696 and the fix is here: [2] https://github.com/Bumblebee-Project/bbswitch/tree/hack-lenovo (scroll down)

I do that and now I can finally open steam using 'optirun' (bumblebee's command). I get a pop-out from Steam telling me to install 'Nvidia experimental driver 310', sadly I get an Ubuntu error from that. I install it via apt-get install and I reboot.

All set and done, until I try to open Counter-Strike. It opens it for 1 second and then instantly closes. It works well when I don't use 'optirun steam' to open steam, but that will just use my integrated card.

So, guys, can you provide me some help ? Remember, try to be as "eli5" as possible, I'm pretty noob to Linux in general.

Edit: This is from my terminal when I try to open Counter-strike

```
AppActive: active "SDL_WINDOWEVENT_SHOWN in 2\n" "SDL_WINDOWEVENT_EXPOSED in 2\n" "SDL_WINDOWEVENT_EXPOSED in 2\n" "SDL_WINDOWEVENT_MOVED in 2\n" "SDL_WINDOWEVENT type 6 (unknown) in 2\n" "SDL_WINDOWEVENT_RESIZED in 2\n" "SDL_WINDOWEVENT_EXPOSED in 2\n" "SDL_WINDOWEVENT_EXPOSED in 2\n" "SDL_WINDOWEVENT_EXPOSED in 2\n" The program 'hl_linux' received an X Window System error. This probably reflects a bug in the program. The error was 'BadDrawable (invalid Pixmap or Window parameter)'. (Details: serial 114 error_code 9 request_code 14 minor_code 0) (Note to programmers: normally, X errors are reported asynchronously; that is, you will receive the error a while after causing it. To debug your program, run it with the --sync command line option to change this behavior. You can then get a meaningful backtrace from your debugger if you break on the gdk_x_error() function.) Missing shutdown function for Sys_InitAuthentication() : Sys_ShutdownAuthentication()

Missing shutdown function for Host_Init( &host_parms ) : Host_Shutdown()

Missing shutdown function for Sys_InitLauncherInterface() : Sys_ShutdownLauncherInterface()

Missing shutdown function for Sys_InitMemory() : Sys_ShutdownMemory()

Missing shutdown function for Sys_Init() : Sys_Shutdown()

Missing shutdown function for FileSystem_Init(basedir, (void *)filesystemFactory) : FileSystem_Shutdown()

Missing shutdown function for Sys_InitArgv( OrigCmd ) : Sys_ShutdownArgv()

Game removed: AppID 10 "Counter-Strike (Beta)", ProcID 7750 saving roaming config store to 'sharedconfig.vdf' roaming config store 2 saved successfully
```

Edit2: After steam told me to install nvidia-experimental-310 it got into conflict with Bumblebee (I could not start Steam anymore using optirun). I found a fix here: [3] http://www.webupd8.org/2012/12/use-nvidia-experimental-drivers-310.html

Now I can start steam again, but I have the same problem as before, Counter-Strike opens for 1 second then quits. I get the same Terminal output.

**Comment by kisak-valve** [comment]
Closing as general video driver quality from 2013 is no longer relevant.
